### ISSUE TITLE
Death Knights in the future can generate with beamswords with invalid focus gems

### ISSUE THREAD

**Issue by demogorgon22** [Chris-plus-alphanumericgibberish/dNAO#2129]
Antimagic rifts and catapsi vortices can end up as the focus gem for their sabers due to the logic added for them. This doesn't display a blade description properly and once they are taken out, cannot go back in. I assume it was meant to implement these as focus gems properly but then it got forgotten?
```
"You now wield an uncursed +6 axiomatic golden beamsword {20}.  You ignite your axiomatic  beamsword."
"B - an uncursed +3 axiomatic  beamsword (lit) (weapon in hand) {20}" 
```

**Comment by Chris-plus-alphanumericgibberish** [comment]
Yes. I believe that it was meant to add the possibility of cancelling or suppressing monsters of the appropriate spell-casting type on hit (since anything being hit by a lightsaber is probably going to die anyway this would have relatively minor balance implications).

**Comment by Chris-plus-alphanumericgibberish** [comment]
Fixed: c2d33f9
